e2e/variantsTest.spec.js

```diff
@@ -96,6 +94,46 @@ import { expect, test } from '@playwright/test'
     variantId:
       'Trojkovka (hodnoty 0=1, 1=2, 2=0; pořadí 1.=>1., 2.=>2., 3.=>3.; A=0, s Ch)',
   },
+  {
+    pageAlt: 'binary',
+    buttons: [
+      '1',
```

Comment:
I think it would be more readable to pass array of digits as one string, e.g. `1010...`.